packages/react/src/components/form/Textfield/Textfield.module.css

```diff
@@ -6,13 +6,74 @@
 .adornment {
   color: var(--fds-semantic-border-neutral-default);
   background: var(--fds-semantic-surface-neutral-subtle);
-  padding: var(--fds-spacing-3);
+  padding: 9px var(--fds-spacing-4);
```

Comment:
Why are we using `9px` here, can we not use `--fds-spacing-2`?

Reply:
Its defined static value in Figma and I couldn't get it to look right with all three using the spacing tokens as its the line-height + x amount of padding, so thought it was best to just let it stay the same for all three.

I have done some changes, so we can test again, but It should also be changed in Figma then :)

Reply:
As we talked on huddle, keep it as this for now. Will update later if we find a better solution